conclave-common/src/main/kotlin/com/r3/conclave/common/EnclaveConstraint.kt

```diff
@@ -146,6 +144,21 @@ class EnclaveConstraint {
         }
     }
 
+    private fun checkSecurityLevel(enclave: EnclaveInstanceInfo) {
+        // If the security level required by the client is insecure than the enclave security should be insecure. This prevents
```

Comment:
I would highlight that enclave security lever _must_ be insecure 
_enclave security should -> enclave security level must_